base/doc/ltnews41.tex

```diff
@@ -466,6 +466,20 @@ \subsection{Process global options once per package}
 %
 \githubissue{1619}
 
+\subsection{Make \cs{label}, \cs{index}, and \cs{glossary} truely invisible in running headers}
+
+\LaTeX{} had a bug since its initial implementation, in that it
```

Comment:
"has had" is the grammatically correct construction here, but I  doubt that anyone cares . . .  these days:-).